stories/nldas3.stories.mdx

```diff
@@ -250,7 +250,7 @@ taxonomy:
           <td>GRACE/GRACE-FO</td>
         </tr>
         <tr>
-          <td>Radar Altimetry Water Level</td>
+          <td>Surface Water Elevation</td>
```

Comment:
Just checking if this change is intended since it is not mentioned in the message.

Reply:
this was intended